File: service-hotel-booking-java/src/main/java/com/acme/services/hotelBooking/java/Worker.java
package com.acme.services.hotelBooking.java;

import io.infinitic.workers.InfiniticWorker;

public class Worker {
    public static void main(String[] args) {
        // infinitic.yml resource is from the dependant "contracts" module
        try (InfiniticWorker worker = InfiniticWorker.fromConfigResource("/infinitic.yml", "/hotelBooking.yml")) {
            worker.start();
        }
    }
}

File: service-hotel-booking-kotlin/src/main/kotlin/com/acme/services/hotelBooking/kotlin/Worker.kt
package com.acme.services.hotelBooking.kotlin

import io.infinitic.workers.InfiniticWorker

fun main() {
    // infinitic.yml resource is from the dependant "contracts" module
    InfiniticWorker.fromConfigResource("/infinitic.yml", "/hotelBooking.yml").use { worker ->
        worker.start()
    }
}
